playground/server/api/regions.ts

```diff
@@ -2,10 +2,7 @@ import { serverMedusaClient } from '#medusa/server'
 
 export default eventHandler(async (event) => {
```

Comment:
question: Is this change intentional? Can we maybe have instead a new playground example of regions fetch instead of replacing the one that will fetch customer on the server side? :)